st2actions/tests/unit/policies/test_concurrency.py

```diff
@@ -248,7 +248,7 @@ def test_on_cancellation(self):
         # Since states are being processed asynchronously, wait for the
         # liveactions to go into scheduled states.
         for i in range(0, 100):
-            eventlet.sleep(2)
```

Comment:
This was actually one because concurrency tests were failing, but if they are not failing anymore, it's fine.

That failure was probably related to Circle CI since it was failing on master as well, before any other stuff was merged.